.github/workflows/obsolete.yaml

```diff
@@ -0,0 +1,36 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues obsolete
+
+on:
+  schedule:
+  - cron: '0 2 1,15 * *'
+  workflow_dispatch:
+        
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    name: Track Obsolete Issues
+
+    steps:
+    - name: Track stale issues and check if obsolete
+      uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-issue-stale: 29
```

Comment:
Curious - why 29 and not 30?

Reply:
The github actions does not have intermediate step that we need to have. In Agones case, we want to apply stale, then wait for 'x' days, apply obsolete and wait again for 'x' then close.. You will also see that close action is part of only close.yaml and is removed from stale.yaml and obsolete.yaml. As for the days,  if I keep 30 days for obsolete.yaml as well, there is a chance of stale.yaml kicking in before obsolete.yaml due to cron disfunction. If that happens then it will remove the 'stale' label as applying label is relevant activity and we will be in loop. For obsolete.yaml and close.yaml, I have 'only-on' option but not in case of 'stale.yaml' Hope it make sense..